webpack.config.cjs

```diff
@@ -40,4 +40,9 @@ module.exports = {
     ],
   },
   plugins: [new MiniCssExtractPlugin()],
+  devServer: {
+    headers: {
+      "Access-Control-Allow-Origin": "*",
```

Comment:
I don't remember why I added this, but this configuration allows CORS requests to the dev server